src/EditorFeatures/VisualBasicTest/Completion/CompletionProviders/SymbolCompletionProviderTests.vb

```diff
@@ -6536,6 +6536,44 @@ End Class
             VerifyItemExists(text, "y")
         End Sub
 
+        <WorkItem(4136, "https://github.com/dotnet/roslyn/issues/4136")>
+        <Fact, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Sub No__ValueWhenDottingIntoEnum()
```

Comment:
Nit: Test name implies the field is named "__value" when it's actually named "value__" (I couldn't remember which it was and the mismatch between the method name and the ```VerifyItemIsAbsent``` was confusing).